ios/FluentUI/Tooltip/Tooltip.swift

```diff
@@ -236,20 +234,22 @@ open class Tooltip: NSObject, TokenizedControlInternal {
     @objc public var textAlignment: NSTextAlignment = .natural
     /// Whether a tooltip is currently showing.
     @objc public private(set) var isShowing: Bool = false
+    public var anchorView: UIView?
```

Comment:
* Does this need to be `@objc` exposed?
* Does this need to be public, or just internal? I don't see any users outside of Fluent (e.g. the test app).